scripts/README.md

```diff
@@ -0,0 +1,49 @@
+# Install without conda
```

Comment:
This one isnt up to date we've changed a lot of things since. Either just remove it and we make a new one in a follow up or update it. Up to you